### ISSUE TITLE
Apache mpm_prefork wrong for Ubuntu 14.04 LTS

### ISSUE THREAD

**Issue by xavadu** [aws/opsworks-cookbooks#345]
Hello,

I don't know if it affect also to other Ubuntu version.

The recipe write the mpm_prefork configuration to the `apache2.conf` template. It is wrong, apache read this configuration for the module configuration file `.../mods-available/mpm_prefork.conf` and ignore the apache2.conf for this module.

So, customize the apache2 recipe attribute don't affect in nothing to the system, and apache use the mpm_prefork config defaults.

It simple issue take me two days of work to find out.

Regards,
J.

**Comment by rln-fw** [comment]
Also just got bitten by this. Even if you comment out the values in mpm__.conf, just loading the module causes the defaults to activate. To fix, you need to move the mpm_prefork config in apache2 to after the 'IncludeOptional mods-enabled/_.conf' statement (or better yet, move the config completely into mods-available/mpm_*.conf).

**Comment by xavadu** [comment]
yes, that is what i did. After a whole day of research until find the issue.

But this is a huge bug, image all this load-based instance that are begging created and destroyed maybe without use all their potential because this. The default recipe ignore if you override the apache2 attributes values.
